packages/tab/README.md

```diff
@@ -69,17 +69,25 @@ For usage please see [mwc-tab-bar](https://github.com/material-components/materi
 
 ### CSS Custom Properties
```

Comment:
This is great! Very close to what we need. We are following the pattern for CSS Custom Properties found in the [button README](https://github.com/material-components/material-components-web-components/blob/master/packages/button/README.md#css-custom-properties).

But that also means we're following another section in button for [Global CSS Properties](https://github.com/material-components/material-components-web-components/blob/master/packages/button/README.md#global-custom-properties)!

Check that out and see if you can mirror button's documentation :)